src/iconsManifest/iconsGenerator.ts

```diff
@@ -156,6 +171,20 @@ export class IconsGenerator implements models.IIconsGenerator {
           `[${constants.extension.name}] Entrypoint in 'package.json' updated`,
         );
       }
+      // update 'vscode:uninstall' script
+      predicate = (line: string): boolean =>
+        line.includes('"vscode:uninstall"');
+      lineIndex = rawText.findIndex(predicate);
+      if (lineIndex > -1) {
+        rawText[lineIndex] = rawText[lineIndex].replace(
+          manifest.scripts['vscode:uninstall'],
+          uninstallEntryPath,
+        );
+        // eslint-disable-next-line no-console
```

Comment:
I guess after https://github.com/vscode-icons/vscode-icons/pull/2402 has been merged, this line can be removed, can't it?

Reply:
Of course not. This is what dynamically changes the path to the script.

Reply:
If you mean the rule, I guess so.

Reply:
Yeah, I meant the rule. I'm sorry for the misunderstanding. :/

Reply:
It's OK. I tend nowadays to fast read everything due to the situation and missing meanings.